nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -96,7 +97,7 @@ controller_server:
     min_y_velocity_threshold: 0.5
     min_theta_velocity_threshold: 0.001
     progress_checker_plugin: "progress_checker"
-    goal_checker_plugin: "goal_checker"
+    goal_checker_plugins: ["precise_goal_checker", "general_goal_checker"]
```

Comment:
I really like the idea of having multiple here as an example, however since this is `> 1` and the BT nodes don't specify any, all nav requests should fail (and if CI isn't failing, we have a bug) since it wasn't specified. 

We should change this to `goal_checker_plugins: ["general_goal_checker"] #precise_goal_checker"` and comment out the `precise_goal_checker` below. I'd like to keep them in the yaml to make the point but shouldn't be commented in just yet